pkg/tokens/parsers/parser_coingecko_all_tokens.go

```diff
@@ -0,0 +1,77 @@
+package parsers
+
+import (
+	"encoding/json"
+	"slices"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/status-im/go-wallet-sdk/pkg/tokens/types"
+)
+
+// Note:
+// Keeping this parser here, but it doesn't provide decimals (they all will be 0).
+// This parser can be updated to use multicall3 (more in pkg/contracts/multicall3) and fetch the decimals from contracts.
+
+// CoinGeckoAllTokens represents a token in CoinGecko format.
+type CoinGeckoAllTokens struct {
+	ID        string            `json:"id"`
+	Symbol    string            `json:"symbol"`
+	Name      string            `json:"name"`
+	Platforms map[string]string `json:"platforms"`
```

Comment:
shouldn't we be able to group tokens together from this list? It doesn't seem we're setting any crossChainID from the coingecko list